### ISSUE TITLE
[Keyboard Navigation-Project-Makecode Documentation]: Focus is moving to some hidden link showing on left panel on Project documentation page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2795]
**User Experience:**
User who depends on Keyboard with screen reader may get misguided about the links and controls if focus moves to any hidden link on Project Page.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/projects
2. Navigate to left panel controls and verify that focus is navigating to hidden link.

**Actual Result:-**
While navigating the page using tab key Focus is moving to some hidden link showing on left panel on Project page.

**Expected Result:-**
While navigating the page using tab key Focus should not move to any hidden link showing on left panel on Project page

**Note:** 
1-Focus also navigates to hidden links after USB and Streaming links under Hardware in the left panel.
2-This issue exists with Firefox also.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Suggested fix:-**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=10 
 

**Please refer the Attachment:**
[MAS2.4.3_Make code_Projects_Focus hidden links.zip](https://github.com/Microsoft/pxt/files/1248368/MAS2.4.3_Make.code_Projects_Focus.hidden.links.zip)

**Comment by veler** [comment]
Issue duplicated with [https://github.com/Microsoft/pxt/issues/2649](https://github.com/Microsoft/pxt/issues/2649)

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/projects and issue is not fixed now. Hence reactivating the bug

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/projects and issue is not fixed now. Hence reactivating the bug. Attached latest video
[Issue is not fixed.zip](https://github.com/Microsoft/pxt/files/1320122/Issue.is.not.fixed.zip)

**Comment by samelhusseini** [comment]
Please verify again. issue should be fixed.

**Comment by sonimishra** [comment]
Verified on https://makecode.microbit.org/, Issue is fixed as now Focus is not moving to any hidden link showing on left panel on Project documentation page I don't have permissions to ADD/Remove labels. Hence not able to add Closed label.

**Comment by samelhusseini** [comment]
Thanks for verifying. Closing bug as fixed.
